src/extensions.rs

```diff
@@ -270,6 +271,78 @@ impl RatchetTreeExtension {
     }
 }
 
+#[repr(u32)]
+#[cfg_attr(rustfmt, rustfmt_skip)]
```

Comment:
Hm, I don't really like `rustfmt_skip`. I understand that you like the formatting here. But couldn't we write this in a way that's coherent with formatting?